### PR TITLE
Lynn Trickey Done with baseline pull request

### PR DESCRIPTION
**Pull request by ltrickey**
"meaningful pull request description"
Heroku deployment: https://trickey-eats.herokuapp.com/
Trello board: https://trello.com/b/OKHFiUYT/api-muncher

# API Muncher
Congratulations! You're submitting your assignment!

## Comprehension Questions
| Question | Answer    |
| :------------- | :------------- |
| How did you go about exploring the Edamam API, how did you try querying the API? |  |
| Describe your API Wrapper.  How did you decide on the methods you created? |  |
| Describe an edge case or failure case test you wrote for your API Wrapper. |  |
| Explain how VCR aids in testing an API. |  |
| What is the Heroku URL of your deployed application? |  |
| Do you have any recommendations on how we could improve this project for the next cohort? |  |

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# See https://help.github.com/articles/ignoring-files for more about ignoring files.
+#
+# If you find yourself ignoring temporary files generated by your text editor
+# or operating system, you probably want to add a global ignore instead:
+#   git config --global core.excludesfile '~/.gitignore_global'
+
```
Comment (CheezItMan) [inline .gitignore:6]: Remember to add `.env` to your `.gitignore` file, so you don't accidentally share your API Key.

**Review by CheezItMan — CHANGES_REQUESTED** [review]
Just remember to hide your API keys!

File: .gitignore (hunk 1 above)
Comment (ltrickey) [reply .gitignore:6]: @CheezItMan thanks for the feedback!  I added this in a separate branch that was just merged.  I don't think I shared the keys online - at least I can't see them anywhere... for the future, is there a way to delete this stuff from git if we do accidentally add them?  Thanks again!
